examples/src/timestamp_queries/mod.rs

```diff
@@ -456,7 +456,9 @@ mod tests {
                 .features(
                     wgpu::Features::TIMESTAMP_QUERY
                         | wgpu::Features::TIMESTAMP_QUERY_INSIDE_ENCODERS,
-                ),
+                )
+                // see https://github.com/gfx-rs/wgpu/issues/2521
+                .expect_fail(FailureCase::always().flaky()),
```

Comment:
Ack, sorry one more comment, can you add a message to make sure that other issues with the test are caught, `flaky` is a big hammer (ignore status of the test) so we want to refine it as much as possible.

Reply:
I added a "unexpected timestamp" panic message.